docs/api.md

```diff
@@ -698,6 +698,9 @@ sigma = await api.getConfig('sigma')
 [Try yourself in the setConfig example >>](https://imjoy.io/#/app?plugin=oeway/ImJoy-Demo-Plugins:setConfig&w=examples)
 
 ### api.getFileUrl
+
+*** WARNING: `api.getFileUrl` is deprecated since `api_version > 0.6.1`, please use `api.getFileManger` instead.
```

Comment:
Should this be `api_version > 0.1.6`?

Where do we see the latest  `api_version`?